### PR TITLE
doc/cephfs/client-auth.rst: correct ``fs authorize cephfs1 /dir1 client.x rw``

### PR DESCRIPTION
**Pull request by yehaifeng**
doc/cephfs/client-auth.rst: correct ``fs authorize cephfs1 /dir1 client.x rw``

Author: 叶海丰 <769358362@qq.com>
Date:   Fri Dec 15 17:05:38 2023 +0800

    doc/cephfs/client-auth.rst: correct ``fs authorize cephfs1 /dir1 client.x rw``

### CODE REVIEW

File: doc/cephfs/client-auth.rst [hunk 1]
```diff
@@ -316,8 +316,8 @@ Changing rw permissions in caps
 It's not possible to modify caps by running ``fs authorize`` except for the
 case when read/write permissions have to be changed. This so because the
```
Comment (zdover23) [inline doc/cephfs/client-auth.rst:317]: s/This so/This is/

File: doc/cephfs/client-auth.rst [hunk 2]
```diff
@@ -316,8 +316,8 @@ Changing rw permissions in caps
...
 ``fs authorize`` becomes ambiguous. For example, user runs ``fs authorize
```
Comment (zdover23) [inline doc/cephfs/client-auth.rst:318]: This section is missing a number of definite and indefinite articles and will have to be written conditionally.

Comment (vshankar) [reply doc/cephfs/client-auth.rst:318]: @zdover23 I'm merging this change. @yehaifeng addressed your pervious comment and this comments probably needs a doc refactor.

**Review by vshankar — APPROVED** [review]

File: doc/cephfs/client-auth.rst (hunk 2 above)
Comment (zdover23) [reply doc/cephfs/client-auth.rst:318]: @vshankar: Go for it. I'll give it a grammar pass later today.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55246 - Reef backport
https://github.com/ceph/ceph/pull/55247 - Quincy backport

**Pull request merged**